apps/web/src/modules/job/components/Jobform/hooks/useJobForm/schema.ts

```diff
@@ -33,34 +33,41 @@ const shootingSchema = z
 
 const postJobSchema = z
   .object({
-    jobName: z.string({ required_error: 'กรุณากรอกตำแหน่งงาน' }),
-    jobDescription: z.string({ required_error: 'กรุณากรอกรายละเอียดงาน' }),
-    dueDate: z
+    title: z.string({ required_error: 'กรุณากรอกตำแหน่งงาน' }),
+    description: z.string({ required_error: 'กรุณากรอกรายละเอียดงาน' }),
+    applicationDeadline: z
       .instanceof(dayjs as unknown as typeof Dayjs)
       .refine((arg) => arg.isAfter(dayjs()), {
         message: 'วันที่สิ้นสุดการรับสมัครต้องอยู่หลังวันที่ปัจจุบัน',
       }),
     wage: z
-      .string({ required_error: 'กรุณากรอกค่าจ้าง' })
-      .refine((arg) => /^[0-9]*\.?[0-9]*$/.test(arg), {
-        message: 'กรุณาค่าจ้างกรอกเป็นตัวเลข',
-      }),
+      .number({
+        required_error: 'กรุณากรอกค่าจ้าง',
+        invalid_type_error: 'กรุณากรอกค่าจ้างเป็นจำนวนบวก',
+      })
+      .nonnegative({ message: 'กรุณากรอกค่าจ้างเป็นจำนวนบวก' }),
     actorCount: z
-      .string({ required_error: 'กรุณากรอกจำนวนนักแสดง' })
-      .refine((arg) => /^[0-9]*$/.test(arg), {
-        message: 'กรุณากรอกจำนวนนักแสดงเป็นตัวเลขจำนวนเต็ม',
-      }),
+      .number({
+        required_error: 'กรุณากรอกจำนวนนักแสดง',
+        invalid_type_error: 'กรุณากรอกจำนวนนักแสดงเป็นจำนวนเต็มบวก',
+      })
+      .positive('กรุณากรอกจำนวนนักแสดงเป็นจำนวนเต็มบวก')
+      .multipleOf(1, 'กรุณากรอกจำนวนนักแสดงเป็นจำนวนเต็มบวก'),
```

Comment:
Does `positive` not enough in this case ?

Reply:
ต้องเพิ่มอะไรนะ ?

Reply:
อ่ออ มันต้องเป็นจำนวนเต็มป่ะ ไม่งั้นเดี๋ยวมาเป็น 1.5 คน